### PR TITLE
feat: Use Pixi, pyradiomics fork

### PR DESCRIPTION
**Pull request by jjjermiah**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new CI/CD workflow for improved automation of integration and deployment processes.
	- Added support for multiple Python environments and operating systems in testing.
	- Implemented a new package management system using Pixi.

- **Bug Fixes**
	- Updated the workflow to ensure successful execution of jobs based on prior job completions.

- **Documentation**
	- Enhanced `.gitignore` to exclude additional files and directories.

- **Refactor**
	- Restructured `pyproject.toml` to transition from Poetry to Pixi, improving modularity and task management.

- **Chores**
	- Added configuration files for coverage and linting to streamline development processes.

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -1,54 +1,78 @@
...
 
-  Continuous-Deployment:
-    needs: Unit-Testing
-
-    # # Only run this job if new work is pushed to "main"
-    if: github.ref == 'refs/heads/main'
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.12"]
```
Comment (strixy16) [inline .github/workflows/ci-cd.yml:69]: Has this worked? I feel like a lot of packages I've used were with Python 3.9 so I wonder if tests are passing?

**Review by strixy16 — COMMENTED** [review]
Looks good, only worried about Python version in the new ci-cd.yml

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (jjjermiah) [reply .github/workflows/ci-cd.yml:69]: I've extended the testing suite to run all the pytests on python `3.10`, `3.11` and `3.12`
combined with GHA matrix expansion across different machines, all the tests seem to pass. 
image

The `release` step hasnt been tested yet due to not being on `main` though. 

note: cant test yet for `3.13` due to [a bug in scikit image](https://github.com/scikit-image/scikit-image/pull/7448)

File: .github/workflows/ci-cd.yml.disabled [hunk 2]
```diff
@@ -0,0 +1,188 @@
+name: ci-cd
+
+# only run on pushes to main or pull requests
+on:
+  push:
+    branches: ["*"]
+  pull_request:
+    branches: [main]
```
[the remark this answers is not in the document]
Comment (jjjermiah) [reply .github/workflows/ci-cd.yml.disabled:8]: disregard anything related to the `ci-cd.yml.disabled` file

**Pull request merged**